README.md

```diff
@@ -1,12 +1,12 @@
 # Swup CLI
 
-<p class="shields">
+<div class="shields" align="center">
```

Comment:
The centered shields [look a bit weird](https://github.com/swup/cli/tree/feat/prepare-for-docs#readme), as the headline above and the text below aren't centered. It works on the main repo with a centered logo above, but I'd suggest skipping it here. WDYT?

Reply:
Totally! Couldn't debug it locally :)